.idea/.gitignore

```diff
@@ -0,0 +1,5 @@
+# Default ignored files
```

Comment:
I don't think it's a good idea to commit 'Intellij'-specific files here because most of us are running VSCode.

Reply:
> I don't think it's a good idea to commit 'Intellij'-specific files here because most of us are running VSCode.

yeah right, I used Webstorm and it automatically generated the `.idea` file. Should have deleted it when committing.

Reply:
@trangntt-016 do you want to do another PR to add `.idea` to our https://github.com/Seneca-ICTOER/IPC144/blob/main/.gitignore?  Then other students that use it won't have this problem.